src/fetcher/main.rs

```diff
@@ -0,0 +1,146 @@
+// This is free and unencumbered software released into the public domain.
+use asimov_readwise_module::providers::ReadwiseType;
+use clap::Parser;
+
+#[derive(Parser)]
+#[command(name = "asimov-readwise-fetcher")]
+#[command(about = "URL protocol fetcher. Consumes a URL input, produces JSON output.")]
+struct Args {
+    #[arg(value_name = "INPUT-URL")]
+    input_url: String,
+
+    /// Page size (number of items per page, default: fetch all)
+    #[arg(long, value_name = "SIZE")]
+    page_size: Option<usize>,
+
+    /// Specific page number to fetch (1-based, default: fetch all)
+    #[arg(long, value_name = "NUM")]
+    page: Option<usize>,
+}
+
+fn main() -> Result<clientele::SysexitsError, Box<dyn std::error::Error>> {
+    use asimov_module::getenv;
+    use asimov_module::secrecy::ExposeSecret;
+    use asimov_readwise_module::{api::readwise::ReadwiseClient, find_provider_for};
+    use clientele::SysexitsError::*;
+    use std::io::stdout;
+
+    clientele::dotenv().ok();
+
+    let args = Args::parse();
+
+    #[cfg(feature = "tracing")]
+    tracing_subscriber::fmt()
+        .with_writer(std::io::stderr)
+        .with_max_level(tracing_subscriber::filter::LevelFilter::WARN)
+        .init();
```

Comment:
There's an helper in [asimov-module](https://github.com/asimov-platform/asimov.rs/blob/f5ef5ba9ea5e92ace3361c60566ccb049d889646/lib/asimov-module/src/lib.rs#L30-L43) for this. Not all modules have been updated to use it but we're trying to standardize on it. It's in the template [here](https://github.com/asimov-modules/asimov-template-module/blob/ce3b631fc96cebede01879d6d7c53836bee71ac0/src/emitter/main.rs#L41-L43).

Requires adding the [flags field](https://github.com/asimov-modules/asimov-template-module/blob/ce3b631fc96cebede01879d6d7c53836bee71ac0/src/emitter/main.rs#L15-L16) to the `struct Args` that you have.


```suggestion
    #[cfg(feature = "tracing")]
    asimov_module::init_tracing_subscriber(&options.flags).expect("failed to initialize logging");
```